server/src/main.rs

```diff
@@ -2,29 +2,52 @@ mod board;
 mod handler;
 mod user;
 
+use std::io::{Error, ErrorKind};
+
 use actix_cors::Cors;
 
 use actix_web::middleware::Logger;
 use actix_web::{web, App, HttpServer};
 
-use crate::board::Registry;
+use crate::board::{HealthChecker, Registry};
 
-#[actix_web::main]
+#[tokio::main]
```

Comment:
do we need to boot an actor system manually in this case? or does the `HttpServer` do that for us?

Reply:
yep - looks like actix web boots up the actors.